### ISSUE TITLE
Options page is very hard to find

### ISSUE THREAD

**Issue by Juraj-Masiar** [skhzhang/time-based-themes#21]
Hello,
You should automatically open the Options page after installation, because in Firefox it's so so hard to find it.
You can use something like this:
```
browser.runtime.onInstalled.addListener(async ({reason, previousVersion}) => {
  if (reason === 'install') {
    browser.runtime.openOptionsPage()
  }
```
Or add a toolbar icon that will open it.

**Comment by skhzhang** [comment]
Thanks for the feedback! That's a great idea. 

I agree, the Options page has definitely been made more difficult to find in recent versions of Firefox (it was easier years ago).

I'll add this once I get the chance.

**Comment by skhzhang** [comment]
This is fixed in 1.4.0 with c0450bcd3530beeda4a8e5bac37ceb998758d16e. It will open automatically upon install.

Thank you for the suggestion and the code snippet on this. :)
